code/datums/elements/strippable.dm

```diff
@@ -447,7 +447,7 @@
 						return
 
 					// They equipped an item in the meantime
-					if (!isnull(strippable_item.get_item(owner)))
+					if(!isnull(strippable_item.get_item(owner)))
```

Comment:
don't touch this file if you're not doing anything with it other than this

Reply:
Yep, my mistake